lib/solr_wrapper/instance.rb

```diff
@@ -103,35 +103,31 @@ def restart
     ##
     # Check the status of a managed Solr service
     def status
-      return true unless config.managed?
-
-      out = exec('status').read
-      out =~ /running on port #{port}/
+      exec('status').read
     rescue
-      false
+      'No status information available'
```

Comment:
... which only happens in exceptional cases. I'm 👍 to giving users better access to this status information, but if we're going to do that, we should also give them enough information when things are going wrong.